src/encoded/schemas/gene.json

```diff
@@ -122,6 +204,56 @@
             },
             "permission": "import_items"
         },
+        "locations": {
+            "title": "Gene location coordinates",
+            "description": "Genomic locations for different assembles of this gene.",
+            "type": "array",
+            "default": [],
+            "uniqueItems": true,
+            "items": {
+                "title": "Gene Location Coordinate",
+                "description": "Genome position information for one assembly of this gene.",
+                "type": "object",
+                "additionalProperties": false,
+                "required": ["assembly", "chromosome", "start", "end"],
+                "properties": {
+                    "assembly": {
+                        "title": "Mapping assembly",
+                        "description": "The genome assembly to which coordinates relate. e.g. GRCh38.",
+                        "type": "string",
+                        "enum": [
+                            "ce10",
+                            "ce11",
+                            "dm3",
+                            "dm6",
+                            "GRCh38",
+                            "hg19",
+                            "mm10",
+                            "mm9"
+                        ]
+                    },
+                    "chromosome": {
+                        "title": "Chromosome",
+                        "description": "The number (or letter) designation for the chromosome, e.g. chr1 or chrX",
+                        "type": "string",
+                        "pattern": "^(chr[0-9XYMT]+)$"
```

Comment:
I think this regex is not going to work with C.elegans and Drosophila chromosome names. See: https://github.com/ENCODE-DCC/encValData/blob/master/dm3/chrom.sizes and https://github.com/ENCODE-DCC/encValData/blob/master/ce11/chrom.sizes

Reply:
Ok, thanks for point that out. Will change it.